godot-core/src/obj/raw_gd.rs

```diff
@@ -474,18 +479,36 @@ where
         }
     }
 
+    #[cfg(debug_assertions)]
+    pub(crate) fn ensure_storage_not_null(&self) {
+        let binding = self.resolve_instance_ptr();
+        if !binding.is_null() {
+            return;
+        }
+        // Non-tool classes can't be instantiated in the editor.
+        if crate::classes::Engine::singleton().is_editor_hint() {
+            panic!(
+                    "Class {} -- null instance; does the class have a Godot creator function? \
+                    Ensure that the given class is a tool class with #[class(tool)], if it is being accessed in the editor.",
+                    std::any::type_name::<Self>()
+                )
```

Comment:
Formatting here :slightly_smiling_face: 
```suggestion
            panic!(
                "Class {} -- null instance; does the class have a Godot creator function? \
                Ensure that the given class is a tool class with #[class(tool)], if it is being accessed in the editor.",
                std::any::type_name::<Self>()
            )
```